models/banner.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+
+const MODEL = {
+    id: Joi
+        .number()
+        .integer()
+        .positive(),
+
+    message: Joi
+        .string()
+        .max(512)
+        .description('Body of banner to display')
+        .example('Due to planned upgrade of Kubernetes, Screwdriver will be down'),
+
+    isActive: Joi
+        .boolean()
+        .description('Flag if the banner is active')
+        .example(true)
+        .default(false),
+
+    dateCreated: Joi
+        .string()
+        .isoDate()
+        .description('When this banner was created')
+        .example('2017-01-06T01:49:50.384359267Z'),
+
+    createdBy: Joi
+        .number()
+        .integer()
+        .positive()
+        .description('ID of user creating the banner'),
```

Comment:
I don't think we have user ID numbers. It'll have to be username https://github.com/screwdriver-cd/data-schema/blob/be07dfdab5d6e4ede1c46a8bec5dd213ec512e97/models/user.js#L12-L16